pipeline/transform.py

```diff
@@ -1,21 +1,77 @@
 """Transform script for the plants pipeline."""
 import pandas as pd
+from time import perf_counter
 
+MINIMUM_SOIL_MOISTURE = 0
+MAXIMUM_SOIL_MOISTURE = 55
+MINIMUM_TEMPERATURE = 5
+MAXIMUM_TEMPERATURE = 50
 
-def make_botanist_df(df):
 
-    botanist = df[["botanist_name", "botanist_email", "botanist_phone"]]
+def clean_recording_taken_data(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Cleans the recording taken column from the DataFrame and
+    drops any NaN values once values converted to datetimes.
+    """
 
-    return botanist
+    df["recording_taken_bool"] = pd.to_datetime(
+        df["recording_taken"], errors="coerce").notna()
+    df = df.drop(df[df.recording_taken_bool == False].index)
 
+    df = df.drop(columns=["recording_taken_bool"])
```

Comment:
drop all  rows which aren't a datetime format by using a bool, then drop the rows that are False is a good way of doing it